.github/workflows/athena_test.yml

```
@@ -64,25 +72,56 @@
+          # Display individual coverage reports in CI
+          echo "📊 Individual Coverage Reports:"
+          echo "=============================="
+          for module in modules/programming/module_programming_llm modules/text/module_text_llm modules/modeling/module_modeling_llm; do
+            if [ -d "$module/.venv" ]; then
+              module_name=$(echo $module | sed 's|modules/||' | sed 's|/|_|g')
+              echo "📈 $module:"
+              $module/.venv/bin/python -m coverage report --rcfile=.coveragerc --data-file=".coverage.${module_name}" --show-missing || echo "No coverage data available"
+              echo ""
+            fi
+          done
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)